tests/phpunit/tests/post/meta.php

```diff
@@ -227,21 +225,21 @@ function test_update_meta() {
 	 * @see https://core.trac.wordpress.org/ticket/12860
 	 */
 	function test_funky_post_meta() {
-		$classy = new StdClass();
-		$classy->ID = 1;
-		$classy->stringy = "I love slashes\\\\";
-		$funky_meta[] = $classy;
+		$classy          = new StdClass();
+		$classy->ID      = 1;
+		$classy->stringy = 'I love slashes\\\\';
```

Comment:
Is the double quote needed?

Reply:
Single quotes used upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/post/meta.php

Reply:
Both are wrong.

Reply:
I think I was misreading it. My mistake.